### DISCUSSION TITLE
Warning: "Detected long formatted time series but failed to convert from long frame: long series must be sorted ascending by time to be converted. Returning table format instead"

### DISCUSSION THREAD

**Discussion by felixmondelo** [marcusolsson/grafana-gantt-panel#35] [Q&A] [answered]
Hi,

What means the warning "detected long formatted time series but failed to convert from long frame: long series must be sorted ascending by time to be converted. Returning table format instead"? It's on the title always.

![image](https://user-images.githubusercontent.com/14011841/121324244-d2e7d300-c910-11eb-8199-710002894205.png)

**Comment by marcusolsson** [comment, marked as answer]
The warning comes from your data source rather than the Gantt panel. Queries need to be formatted as Table data to be used with the Gantt panel. It seems like the data source query has some issues and falls back to Table data (which is why the visualization still works).
